debian/salsa-ci.yml

```diff
@@ -638,111 +549,6 @@ mariadb.org-10.5 to mariadb-10.5 upgrade:
     variables:
       - $CI_COMMIT_TAG != null && $SALSA_CI_ENABLE_PIPELINE_ON_TAGS !~ /^(1|yes|true)$/
 
-mariadb.org-10.4 to mariadb-10.5 upgrade:
```

Comment:
Are these duplicates or why these are removed?

Reply:
Updated version pushed to https://salsa.debian.org/mariadb-team/mariadb-server/-/pipelines/356697 - will check tomorrow if it makes sense to have these old upgrade tests.